contents/blog/posthog-marketing.md

```diff
@@ -0,0 +1,219 @@
+
+Overall, pretty well. Most pleasingly, we smashed our target 420 weekly new org. signups – we consistently hit over 450 per week in August and had our first 500+ week in September.
+
+Progress on our organic SEO target (5,000 weekly users) has been steady rather than spectacular. We were at ~4,000 per week in July and have been hovering around the 4,700 mark until recently, where we hit 5,200 in a week. Time will tell if this trend will hold.
+
+We achieved 14 of the 20 G2 profile reviews we aimed for in Q3 with an average score of 4.4. Not bad.
+
+## What's next?
+
+We're currently reviewing our objectives for Q4, but we're also looking to make improvements to what and how we track our marketing efforts.
+
+### Quality signals for new users
+
+We're currently working on a process for "quality scoring" signups. 
+
+Our favored solution is based on an Ideal Customer Profile (ICP) score created by our customer success team. Once we've got the process nailed, it'll give us a much better picture of which channels are most valuable to us. 
+
+We're also considering several other supporting signals, such as new users who ingest events or invite colleagues. We're still debating the best approach here.
+
+### Improving how we track attribution
+
+Until recently, we weren't focused on attribution outside of UTM parameters on our paid marketing activities. Marketing attribution is notoriously hard, so we felt our time and energy was better spent elsewhere. That said, it's becoming more important as we scale.
+
+We recently started collecting more information from users at signup – we've added an optional text field where people can tell us where they heard about us. We collect this data in PostHog, and manually export it for analysis. It's not the most automated process ever, but it's a start and something we'll evolve over time. 
+
+### Marketing experiments
+
+Thus far, we've focused on content and developing marketing channels that work. Eventually, we'll need to spend more time on website optimization and experimentation. We're not there yet, and we're wary of over optimization, but we have all the tools we need when we need them. 
+
+## Final thoughts (aka why we use PostHog in the marketing team)
+
+In summary, there are a few important reasons why we use PostHog instead of traditional marketing tools:
+
+1. **It does (almost) everything already:** Outside of a few edge cases, PostHog already does 95% of what we need. If (and when) we need more, we'll make the case for building that functionality, rather than resorting to some other tool.
+
+2. **We're interested in outcomes, not activity:** A lot of traditional web metrics aren't useful. We want to know "how does x impact y", not just "how many people did x or y". Funnel insights are huge part of this, and the ability to go from viewing a funnel to watching session recordings of real users is just as powerful for us as it is for our product teams.
```

Comment:
```suggestion
2. **We're interested in outcomes, not activity:** We want to know "how does x impact y" and why, not just "how many people did x or y". This is where traditional marketing tools fall short. No marketing platform lets you go from viewing a funnel to watching session recordings of real users on your website.
```

@ivanagas Reworded this a little. I feel like we're saying what you're suggesting without explicity saying it, if that makes sense? In any case, I think this wording is better.